bin/serve.ts

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env node
+import { argv } from 'yargs';
+import { existsSync, readFileSync } from 'fs'
+import { TestxServer } from '../src';
+
+async function startServer(model: string) {
+    const server = new TestxServer({
+        schema: model
+    });
+
+    // start the server
+    await server.start();
+    // retrieve the server url
+    console.log(`Running on ${await server.httpUrl()}`);
+}
+
+
+if (argv.model) {
+    const modelLocation = argv.model as string;
+    if (existsSync(modelLocation)) {
+        const fileContent = readFileSync(modelLocation, { encoding: 'utf8' });
```

Comment:
What about moving the file read part inside Testx itself?
https://github.com/aerogear/graphql-testx/issues/97